pkg/generators/forwarding/fluentd/syslog_conf.go

```diff
@@ -6,3 +6,9 @@ func (conf *outputLabelConf) SyslogPlugin() string {
 	}
 	return "syslog_buffered"
 }
+
+func (conf *outputLabelConf) Rfc() string {
+	//return "rfc3164"
+	// TODO update it whem merged with new v1 api
+	return "rfc5424"
```

Comment:
have tested with both "rfc5424" and "rfc3164", and both tests pass with changes in https://github.com/openshift/origin-aggregated-logging/pull/1874